http_service/Dockerfile.bg_worker

```diff
@@ -14,6 +14,7 @@ WORKDIR /code/
 ARG CHECK_MODELS
 ENV CHECK_MODELS="${CHECK_MODELS}"
 
-RUN bash /code/http_service/ensure_models.sh
+RUN bash /code/http_service/tools/ensure_models.sh
+RUN bash /code/http_service/tools/bootstrap-mercurial.sh
```

Comment:
Maybe you could base this image off the mozilla/bugbug-commit-retrieval image instead, so it will be quicker to build

Reply:
Done.